pages/linux/sbcast.md

```diff
@@ -0,0 +1,13 @@
+# sbcast
+
+> Send a file to a job's allocated nodes.
+> This command should only be used from within a Slurm batch job.
+> More information: <https://slurm.schedmd.com/sbcast.html>.
+
+- Send a file to all nodes allocated to the current job:
+
+`sbcast {{path/to/file}} {{path/to/destination}}`
+
+- Autodetect shared libraries the transmitted file depends upon and transmit them as well:
+
+`sbcast --send-libs={{yes}} {{path/to/executable}} {{path/to/destination}}`
```

Comment:
Do you have a link to documentation on this? All the documentation I've seen for `sbcast` doesn't have `--send-libs` as an argument.

Reply:
https://slurm.schedmd.com/sbcast.html#SECTION_OPTIONS